assets/queries/crossplane/aws/db_instance_storage_not_encrypted/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "e50eb68a-a4af-4048-8bbe-8ec324421469",
+  "queryName": "DB Instance Storage Not Encrypted",
+  "severity": "HIGH",
+  "category": "Encryption",
+  "descriptionText": "The parameter storage_encrypted in aws_db_instance must be set to 'true' (the default is 'false').",
```

Please, improve the description

```suggestion
"descriptionText": "The parameter storageEncrypted in RDSInstance must be set to 'true' (the default is 'false').",
```